### ISSUE TITLE
Can we prevent dragging in vertical Direction ?

### ISSUE THREAD

**Issue by riyaistar** [SortableJS/jquery-sortablejs#4]
Below is [https://jsfiddle.net/riyamitra/4oatsyde/] (js fiddle for the scenario) I need to prevent vertically  scrolling as my list horizontally scrolling. I want the dragging element to be only in horizontal direction.

**Comment by depuits** [comment]
I don't think you can currently limit the position of the dragged object. As an alternative you can easely hide it by using the `dragClass`. https://jsfiddle.net/jq49wr0t/
